docs/content/how-tos/rules/FL0075.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FL0075
+category: how-to
+hide_menu: true
+---
+
+# VariableLengthGreaterThanOne (FL0075)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Rule to detect usage of 1 letter as variable names, function names, and parameter names.
+
+## Rationale
+
+Variable name represents the information the variable contains. It helps to improve code quality particularly during debugging. Variable length of more than one makes reading code pretty easier.
+
+## How To Fix
+
+Make sure that the length of all your variable names, function names, and parameter names are up 2 letters.
+
+## Rule Settings
+
+    {
+        "variableLengthGreaterThanOne": {
+           "enabled": true
```

Comment:
let's make this enabled property consistent with fsharplint.json

Reply:
Done.